lib/src/dart3_suggestors/required_props/bin/codemod.dart

```diff
@@ -148,6 +149,16 @@ class CodemodCommand extends Command {
           parsedArgs.argValueAsNumber(_Options.publicMaxAllowedSkipRate),
     );
 
+    exitCode = await runInteractiveCodemodSequence(
+      dartPaths,
+      [
+        ClassComponentRequiredDefaultPropsMigrator(null, recommender),
```

Comment:
Does this not work when added to the sequence below, like so?
```diff
     exitCode = await runInteractiveCodemodSequence(
       dartPaths,
       [
+        ClassComponentRequiredDefaultPropsMigrator(null, recommender),
         RequiredPropsSuggestor(
           recommender,
           trustRequiredAnnotations:
               parsedArgs[_Flags.trustRequiredAnnotations] as bool,
         ),
       ],
       defaultYes: true,
       args: codemodArgs,
       additionalHelpOutput: argParser.usage,
     );
```

Not a huge deal, but it would be nice if consumers who wanted to apply all patches via the prompt (like if they didn't think to pass `--yes-to-all`) didn't have to do so twice.

Reply:
It combines the patches when I put them together 😢 

```
'mixin TestPrivatePropsMixin on UiProps {\n'
              '  String/*?*/ notDefaultedOptional;\n'
              '  /*late*/ String notDefaultedAlwaysSet;\n'
              '  /*late/*?*/*/ String/*?*/ defaultedN/*?*/ullable;\n'
              '  /*late*/ num/*!*/ defaultedNonNullable;\n'
              '}\n'
```
https://github.com/Workiva/over_react_codemod/actions/runs/11372701838/job/31637614201?pr=299

Reply:
Hm okay, thanks for trying!

There's probably a bug in dart_codemod, then, possibly due to reusing [FileContext/AnalysisContextCollection](https://github.com/Workiva/dart_codemod/blob/ac038c4ec6d57c15563e990d54fc87af8020135c/lib/src/run_interactive_codemod.dart#L214-L217) between suggestors.